doc/reference/eventengine.rst

```diff
@@ -150,9 +150,12 @@ Available ``event_type`` and ``alert_type`` values
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Two of the available alert attributes that can be matched against in severity
-rules are ``event_type`` and ``alert_type``. See the :doc:`event- and
-alert-type reference documentation <alerttypes>` for a detailed list of
-available type names to match.
+rules are ``event_type`` and ``alert_type``. However, ``event_type`` is a
+Python object: To match against an event type id/name, you must match against
```

Comment:
Would it be worth it to turn it into a database table instead or too much hassle? Not that it affects this PR at all.

Reply:
Well, it is actually already a Django model instance, `nav.models.event.EventType`, which is why you can't match it against a string.